tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -2180,9 +2182,11 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		time.Sleep(time.Duration(pandoraSyncWaitTime) * time.Second)
 
 		defer func() {
-			ginkgo.By("Deleting FCD")
-			err := e2eVSphere.deleteFCD(ctx, fcdID, defaultDatastore.Reference())
-			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			if !supervisorCluster {
```

Comment:
Can you please explain why this if condition is required? We should delete FCD in case of supervisor cluster as well.